src/runtime/composables/local/useAuth.ts

```diff
@@ -105,7 +105,7 @@ const getSession: GetSessionFunc<SessionData | null | void> = async (getSessionO
   return data.value
 }
 
-const signUp = async (credentials: Credentials, signInOptions?: SecondarySignInOptions) => {
+const signUp = async (credentials: Credentials, signInOptions?: SecondarySignInOptions, registerOptions?: RegisterOptions) => {
```

Comment:
Sorry to bug you one last time, but lets rename all of the options from `registerOptions` to `signUpOptions` to be more inline with the function name and then we are good to go 🥳